### PR TITLE
Rename package

### PR DESCRIPTION
**Pull request by isaac-tfn**
Describe Changes
---

Task Checklist
----
- [ ] Have unittests been added (testing individual functions and their edge cases)
- [ ] Have integration tests been added (if applicable, to test modules of functionality)
- [ ] Updated RELEASE.md to reflect changes in PR
- [ ] Have new dependencies been added?
  - [ ] Have they been added to either `requirements.txt` or `requirements_dev.txt`.
  - [ ] Have they been added to `setup.cfg`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ![Transport for the North Logo](https://github.com/Transport-for-the-North/caf.toolkit/blob/main/docs/TFN_Landscape_Colour_CMYK.png)
 
-<h1 align="center">CAF core</h1>
+<h1 align="center">CAF base</h1>
 
 [//]: # (Use the below and update URLs to add more badges when deploying)
 
-[//]: # (<a href="https://pypi.org/project/caf.toolkit/"><img alt="Supported Python versions" src="https://img.shields.io/pypi/pyversions/caf.package.svg?style=flat-square"></a>)
+[//]: # (<a href="https://pypi.org/project/caf.base/"><img alt="Supported Python versions" src="https://img.shields.io/pypi/pyversions/caf.package.svg?style=flat-square"></a>)
```
Comment (MattBuckley-TfN) [inline README.md:7]:
```suggestion
[//]: # (<a href="https://pypi.org/project/caf.base/"><img alt="Supported Python versions" src="https://img.shields.io/pypi/pyversions/caf.base.svg?style=flat-square"></a>)
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-[//]: # (<a href="https://pypi.org/project/caf.toolkit/"><img alt="Latest release" src="https://img.shields.io/github/release/transport-for-the-north/caf.package.svg?style=flat-square&maxAge=86400"></a>)
+[//]: # (<a href="https://pypi.org/project/caf.base/"><img alt="Latest release" src="https://img.shields.io/github/release/transport-for-the-north/caf.package.svg?style=flat-square&maxAge=86400"></a>)
```
Comment (MattBuckley-TfN) [inline README.md:9]:
```suggestion
[//]: # (<a href="https://pypi.org/project/caf.base/"><img alt="Latest release" src="https://img.shields.io/github/release/transport-for-the-north/caf.base.svg?style=flat-square&maxAge=86400"></a>)
```

**Review by MattBuckley-TfN — CHANGES_REQUESTED** [review]
Looks good, just some links in the README need updating (I've added suggestions) and the link to releases in RELEASE.md (line 16).

**Comment by MattBuckley-TfN** [comment]
Fixes #61
